application/controllers/admin/tokens.php

```diff
@@ -2769,4 +2796,70 @@ protected function tokenIsSetInEmailCache($iSurveyId, $tid, $bIsInvitation)
             && isset($_SESSION[$cacheName][$sType][$tid])
             && $_SESSION[$cacheName][$sType][$tid] > 0;
     }
+
+    /**
+     * Dispatches the beforeTokenImport event
+     * @param int $iSurveyId
+     * @param array $aToken
+     * @param string $sUploadCharset
+     * @param boolean $bFilterDuplicateToken
+     * @param boolean $bFilterBlankEmail
+     * @param boolean $bAllowInvalidEmail
+     * @param array $aFilterDuplicateFields
+     * @param string $sSeparator
+     * @param boolean $bShowWarningToken
+     * @param int $iRecordCount
+     * @return array
+     */
+    protected function dispatchBeforeTokenImport($iSurveyId, $aToken, $iRecordCount, $aOptions)
```

Comment:
I'm sorry, but now PHPDocs is not correct anymore. :laughing: Just replace options with `array $aOptions`.

Reply:
Updated. :)